nnpdf_data/nnpdf_data/theory.py

```diff
@@ -0,0 +1,51 @@
+"""
+This module provides an unique source and definition for all the possible parameters
+that a theory card can contain. 
+It also implement some utilities function to manage the theory cards. 
+"""
+
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class TheoryCard:
+    ID: int # ID number of the theory
+    PTO: int # Perturbative order (0 = LO, 1 = NLO, 2 = NNLO ...)
+    FNS: str # Flavor number scheme (i.e. FONLL-C)
+    DAMP: int # Whether a damping function is applied or not for FONLL
+    DAMPPOWERc: int = None # Power of the damping factor in FONLL for the c
+    DAMPPOWERb: int = None # Power of the damping factor in FONLL for the b
```

Comment:
All the optional ones need to go to the end (or, alternatively, we can add a default to every option)